bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/utils/Messages.java

```diff
@@ -295,6 +295,7 @@ public class Messages extends NLS {
 	public static String resources_updating;
 	public static String resources_updatingEncoding;
 	public static String resources_workspaceClosed;
```

Comment:
Do we still use this message in other places?

Reply:
Yes it is used in org.eclipse.core.internal.resources.Workspace.prepareOperation(ISchedulingRule, IProgressMonitor)